README.md

```diff
@@ -22,18 +18,140 @@ All `.md` files within the `src/` folder will be treated as something that relat
     - `_section.md` - (required) Metadata that describes the section for Zendesk
     - `{article}.md` - The article that will be sent to Zendesk. It also contains metadata for the article
 
-# Local development
+## Metadata
 
-TODO: Update this section
+There are some required and optional fields for each of the metadata files. The required fields are used to reference the Zendesk object, while the optional fields are used to populate the content or metadata for the object.
 
-- Clone this repo into a docker container volume
-- Create a `.env` file at the root of the project, copying the contents of `.env.example`, replacing the values with your own
-- Run `npm run start` to start the Zendesk theme compiler and the preview server
+### Category (`_category.md`)
+
+```yaml
+---
+zendesk:
+  category_id: 24638797677853 # Required. The ID of the category in Zendesk
+  name: Home Assistant Green # Required. The name of the category in Zendesk
+  position: 1 # Optional. Defaults to 0
+  description: The affordable Home Assistant Green is the... # Optional. The description of the category in Zendesk
+---
+```
+
+### Section (`_section.md`)
+
+```yaml
+---
+zendesk:
+  section_id: 24737097675549 # Required. The ID of the section in Zendesk
+  name: Getting Started # Required. The name of the section in Zendesk
+  position: 1 # Optional. The position of the section in Zendesk in relation to other sections
+  description: All the things to get you started... # Optional. The description of the section in Zendesk
+---
+```
+
+### Article (`_<article-slug>.md`)
+
+```yaml
+---
+zendesk:
+  article_id: 24737667232413 # Required. The ID of the article in Zendesk
+  name: Getting started with Home Assistant Green # Required. The name of the article in Zendesk
+  position: 2 # Optional. The position of the article in Zendesk in relation to other articles
+---
+```
+
+# Creating / deleting content
+
+This repo does not create or delete content in Zendesk. It only updates the content of existing articles, sections, and categories. There are a few things to note:
+
+## Creating content
+
+If you want to create content, you must first make the content structure in Zendesk. This includes creating the category, section, and article(s). You do not need to populate any content in these objects. Once created, you must obtain the ID for each object. The easiest way to do this is to "preview" the article/section/category in Zendesk. The ID will be in the URL. For example, if the URL is `https://support.nabucasa.com/hc/en-us/articles/24737667232413-Setting-up-the-device`, the ID is `24737667232413`. You can then use this ID in the metadata for the object.
+
+## Deleting content
+
+This repo does not delete content in Zendesk. If you want to delete content, you must do this in Zendesk. Once deleted, remove the relevant files from this repo to avoid errors.
```

Comment:
```suggestion
This repo can delete content of an article in Zendesk, but not an entire article, section, or category. If you want to delete these, you must do this in Zendesk. Once deleted, remove the relevant files from this repo to avoid errors.
```